Rim-Gnoblins Patch

ViralReaction opened the pull request:
## Additions

Describe new functionality added by your code, e.g.
- Patches for Toxic Spear and Grenades
- Patch for Drugs
- Patch pawnkinds so to spawn with ammo

## Reasoning

Why did you choose to implement things this way, e.g.
- Dreg given suppressability equal to psycho due to being unrefined
- Shard given suppressability equal to yayo due to being more refined

## Testing

Check tests you have performed:
- [ ] Compiles without warnings
- [ ] Game runs without errors
- [ ] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

ViralReaction commented:
Patches appear to missing from the PR downloads

N7Huntsman reviewed (changes requested):
Generally looks to be in good shape, but it still needs the scenario patched to have ammo added for the bows, and probably increase the number of tox pots it starts with since they're single-use now--may also be appropriate to switch the texture to use the projectile as the weapon texture, so it doesn't appear as an entire belt.

ViralReaction commented:
Updated scenario to include 100 stone arrows, equal to the CE tribal start. Gave 10 toxic pots to starting, they are pretty innocuous and takes over 30 of them to kill pawns even in Vanilla.

Changed grenade graphic as requested.
